docs/pipelines.md

```diff
@@ -107,34 +106,39 @@ When writing and running your pipeline, note that:
 
-The framework executes your scripts using Docker images which have been preloaded with a fixed set of libraries. These Docker images have yet to be optimised; if you have skills in creating Dockerfiles and would like to help, get in touch!
+The framework executes your scripts using Docker images which have been preloaded with a fixed set of libraries. 
+These Docker images have yet to be optimised; if you have skills in creating Dockerfiles and would like to help, get in touch!
 
 ### Stata
 
 We currently package version 16.1, with `datacheck`, `safetab`, and `safecount` libraries installed; when installed, new libraries will appear [in the stata-docker Github repository](https://github.com/opensafely/stata-docker/tree/master/libraries).
 
-As Stata is a commercial product, and the image incorporates our license key, the Docker image and [Github repository](https://github.com/opensafely/stata-docker/) are currently private, pending work to separate the license from the image.  Get in touch if you need to apply your own license and we can accelerate this work.
+As Stata is a commercial product, and the image incorporates our license key, the Docker image and [Github repository](https://github.com/opensafely/stata-docker/) are currently private, pending work to separate the license from the image.
+Get in touch if you need to apply your own license and we can accelerate this work.
 
 ### Python
 
 The docker image provided is Python 3.8, with [this list of packages installed](https://github.com/opensafely/python-docker/blob/main/requirements.txt).
 
 ### R
 
-The R image provided is R 4.0, with [this list of libraries installed](https://github.com/opensafely/r-docker/blob/master/Dockerfile#L30). Currently our configuration doesn't list the precise versions of the libraries. If you need this, get in touch and we will accelerate this work.
+The R image provided is R 4.0, with [this list of libraries installed](https://github.com/opensafely/r-docker/blob/master/Dockerfile#L30). 
+Currently our configuration doesn't list the precise versions of the libraries. 
+If you need this, get in touch and we will accelerate this work.
 
 
 ## Running your code locally
+
 Whilst you can develop and run code locally using your own installations of R, Stata or Python, it's important to check that these will also successfully run on the real data in an identical execution environment.
 
-The `cohortextractor run` command will execute one or more actions according to the `project.yaml`.
-To see its options, type `cohortextractor run --help`.
+The `opensafely run` command will execute one or more actions according to the `project.yaml`.
+To see its options, type `opensafely run --help`.
 
-For `cohortextractor run` to work:
+For `opensafely run` to work:
 
-* `cohortextractor` version `1.6.1` or higher must be installed.
+* Python version `1.7` or higher must be installed.
```

Comment:
3.7

Reply:
Although perhaps we don't need to bother saying this at all; installing `opensafely` _requires_ python 3.7 and you should get an error message to this effect. If we do want to say this, it probably belongs in the "installation" section rather than here

Reply:
Good spot. I think it's helpful to have the requirements listed explicitly here for completeness and in anticipation of a) people who browse the docs and jump to this page first b) as a prompt for existing users whose system may have changed and who come to this page first looking for answers. 

But yes I agree -- I've made it a bit briefer and updated the python installation pages.